low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -107,8 +105,8 @@ def main(product):
             section=SECTION,
             key='secondary_rabbitmq_server'
             )
-        _check_rabbitmq_status()
-        _start_rabbitmq()
+    _check_rabbitmq_status()
```

Comment:
why rmq initialization made without condition now?

Reply:
Earlier its under the condition of product only in 'cs_legacy_products'. 
So this block won't be executed.
vhost can't be created if rabbitmq is down. So I brought them out as it is needed before creating vhost.

Reply:
When rabbitmq is up, then vhosts would be created right, which is only when sspl can use it anyways.

Reply:
PROD: In deployment time, SSPL is provisioned after rabbitmq setup. So its already active.(LDR_R1)
In DEV env, sspl_post_install->rabbitmq_setup will start the service before sspl_config->sspl_rabbitmq_reinit.

Now, in dev/self provisioning prereq script, we setup rabbitmq as we do in PROD.

Reply:
alright, lets do it